src/test/java/org/openrewrite/kotlin/KotlinTypeMappingTest.java

```diff
@@ -343,7 +343,7 @@ void javaLangObject() {
         JavaType.Class c = (JavaType.Class) firstMethodParameter("javaType");
         assertThat(c.getFullyQualifiedName()).isEqualTo("java.lang.Object");
         assertThat(c.getSupertype()).isNull();
-        assertThat(c.getMethods()).hasSize(12);
+        assertThat(c.getMethods()).hasSize(13);
```

Comment:
Guess I'm running a different Java version that disagrees with CI.
```suggestion
        assertThat(c.getMethods()).hasSize(12);
```